woocommerce/woocommerce
# [e2e-utils-playwright] Added unit tests for package

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/52956

Added unit tests configuration and implemented 3 tests to start with.
Updated the tested function to not throw an error.

### How to test the changes in this Pull Request:

Run the unit tests locally. 3 tests should run and pass.

```
pnpm --filter="@woocommerce/e2e-utils-playwright" test
```

Check CI. A new job running the tests for the package should run and be successful.
